playground/AspireEventHub/EventHubsApi/Program.cs

```diff
@@ -1,14 +1,19 @@
+//#define AZCLI
+#if AZCLI
+using Azure.Identity;
+#endif
 using Azure.Messaging.EventHubs;
 using Azure.Messaging.EventHubs.Producer;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
 
-builder.AddAzureEventHubProducerClient("eventhubns", settings =>
-{
-    settings.EventHubName = "hub";
-});
+builder.AddAzureEventHubProducerClient("eventhubns"
+#if AZCLI
+    , settings => settings.Credential = new AzureCliCredential()
```

Comment:
Why doesn't the DeafultAzureCredential work?

Reply:
I was getting tired of adding/removing this for my own scenario -- not everyone uses the same account on their laptop to access azure. I thought I could get away with leaving it as an "example" while serving my own selfish purposes. If you want me to remove it, I can.

Reply:
Yes, let's remove this clutter from the playground apps.

For this case, we have a vision of being able to define this via config (I'm not sure we've written this up publicly yet) - in your case you would use User Secrets to specify that you want to use AzureCliCredential instead of the DefaultAzureCredential. However, that vision hasn't been realized yet. We are trying to design something with the Azure SDK team that works for everyone. [[FEATURE REQ] Allow Microsoft.Extensions.Azure to select more TokenCredentials (Azure/azure-sdk-for-net#47927)](https://github.com/Azure/azure-sdk-for-net/issues/47927) is related to this.

Reply:
Done.